hw_2/src/main/java/com/epam/spring/homework2/beans/BeanF.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.spring.homework2.beans;
+
+import com.epam.spring.homework2.validator.MyValidator;
+
+public class BeanF implements MyValidator {
+
+    private String name;
+    private double value;
+
+    public BeanF(String name, double value) {
+        this.name = name;
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return "BeanF{" +
+                "name='" + name + '\'' +
+                ", value=" + value +
+                '}';
+    }
+
+    @Override
+    public void validate() {
+        if (name == null || name.isEmpty()) {
```

Comment:
This is not really correct to replace invalid values. Could you please print in the console, some message like `bean [beanName] is invalid` in case if it's invalid.

Also you could combine the condition to smth like 
`if (name == null || name.isEmpty() || value < 0)`

Reply:
fixed